### ISSUE TITLE
Sugestão para economia de tempo e recursos de TIc / sahana

### ISSUE THREAD

**Issue by crlsgms** [dieegom/brumadinho_location#51]
Olá a todos!

Muito interessante o projeto, mas gostaria de deixar a sugestão do software https://github.com/sahana/eden/

já utilizado em larga escala pelo mundo em grandes desastres. Algum motivo de não utilizar / melhorar a ferramenta já existente e recriar uma nova do zero, ainda mais em uma situação tão critica?

Caso precisem posso disponibilizar o servidor e recursos para hospedagem.

**Comment by lucasalvescm** [comment]
Verdade @crlsgms. Seria uma grande economia de tempo e até mesmo contribuir para esse projeto que já existe.

**Comment by maedabr** [comment]
Os últimos commits desse projeto são do dia 25 de janeiro (4 dias atrás). Pelo menos é um projeto vivo ainda.

@crlsgms , já usou esse software? Sabe se ele tem como funcionalidade a previsão de localização das vítimas em desastres como o de brumadinho? Se não tiver, talvez faça mais sentido converter o projeto daqui pra uma extensão do sahana.

**Comment by crlsgms** [comment]
Sim, ele já trabalha com mapas e geolocalização, algumas das funcionalidades discutidas aqui nesta nova aplicação já existem no sahana.

Estou subindo uma instancia aqui para homologar, entrei em contato com a defesa civil para trabalhar em conjunto com eles.

Tentei acionar o grupo deste projeto do diego no telegrm, mas o link de convite ainda estava com o hash sugerido e já expirou.

Caso tenham interesse em testar, devo terminar de subir ainda agora antes do almoço.
Se possivel, me adicionem no grupo do telegram - @crlsgms

**Comment by vaniaon** [comment]
@crlsgms bem interessante! Envie o link para testes depois. Podemos analisar como fazer a integração e oq mais precisa ser feito.

Solicitei no grupo a criação de um novo link, assim que tiver atualizo o readme e aviso aqui.

**Comment by maedabr** [comment]
Legal, também quero entrar no grupo do telegram

**Comment by nibri10** [comment]
Gostaria de entrar no grupo também do telegram

**Comment by crlsgms** [comment]
Criei um grupo do sahana no telegram, caso tenham interesse.

https://t.me/sahanafoss

**Comment by EuFreela** [comment]
Se não manter o projeto no git o intúito de "ajuda" fica descentralizada. Seria legal se mantesse ele aqui ao invés do telegram.

**Comment by vaniaon** [comment]
Pessoal, aqui o link para o grupo no telegram:

https://t.me/joinchat/K2pTZk1Xjo0UEzmCzkJvXQ

**Comment by ypedroo** [comment]
> Olá a todos!
> 
> Muito interessante o projeto, mas gostaria de deixar a sugestão do software https://github.com/sahana/eden/
> 
> já utilizado em larga escala pelo mundo em grandes desastres. Algum motivo de não utilizar / melhorar a ferramenta já existente e recriar uma nova do zero, ainda mais em uma situação tão critica?
> 
> Caso precisem posso disponibilizar o servidor e recursos para hospedagem.


#13 
>Hello everyone!
> 
>Interesting project but i would like to make a sugestion of the software:
https://github.com/sahana/eden/
> 
>Already used in large scale all over the world in big disasters. Any reason for not using it / improve the existing tool and make a new one from scratch, even more in a such critical situation?
>
> In case of need i can provide server and resources for hosting.

**Comment by crlsgms** [comment]
> Se não manter o projeto no git o intuito de "ajuda" fica descentralizada. Seria legal se mantesse ele aqui ao invés do telegram: são 112 forks até o momento do projeto enquanto 7 membros no telegram. A menos que isso seja o objetivo.

O grupo que mandei o link seria para os testes e análise de aderência do sahana, não deste projetodo dieegom, mas imagino que em breve colocarão o link no readme deste projeto.

caso queiram, tem o demo do sahana para análise, e terminei de subir uma instancia de testes.

http://demo.eden.sahanafoundation.org/eden/

https://bruma.franca.unesp.br/

**Comment by acsilveira** [comment]
Hi!
  Congrats all for this brave initiative.

  I would like to share just some thinkings:

  Regarding having a functional version delivered in an useful timing, the idea of adapt some project already used in similar previous disasters like Brumadinho dam, as mentioned by @crlsgms, sounds good.

About the modelling to estimate position of victims after flow (and about https://github.com/sosbrumadinho/brumadinho_location#ideas-to-be-implemented):
I'm not specialist in this field and maybe this community has already defined this point, but a debris-flow model seems to be more suitable to this case considering the mechanics of solid + liquid materials. Some models focus in overall metrics like covered area and flow rate instead allow output with specific throw given a start position. I didn't study which kind of output this model enables but maybe it could be considered:
+ http://swmath.org/software/21531
+ https://github.com/dlgeorge/D-Claw

Also Ushahidi team can give us important important advices:
+ https://github.com/ushahidi
+ https://www.ushahidi.com/

Thank you for your effort.
